pkg/addons/data/ConfigMap/openshift-node/node-config-compute.yaml

```diff
@@ -21,42 +27,40 @@ data:
       - /etc/origin/node/bootstrap.kubeconfig
       cert-dir:
       - /etc/origin/node/certificates
-      cloud-config:
-      - /etc/origin/cloudprovider/azure.conf
-      cloud-provider:
-      - azure
+      container-runtime:
+      - remote
+      container-runtime-endpoint:
+      - /var/run/crio/crio.sock
```

Comment:
This is going to move us to crio.  We should probably pass this along to the rest of the team.

Reply:
Will create separate PR in ansible code for this. have in my local code